packages/subiquity_client/lib/subiquity_client.dart

```diff
@@ -2,32 +2,43 @@ library subiquity_client;
 
 import 'dart:convert';
 import 'package:http/http.dart';
+import 'package:logger/logger.dart';
 import 'src/http_unix_client.dart';
 import 'src/types.dart';
 
 export 'src/types.dart';
 
+/// @internal
+final log = Logger('subiquity_client');
+
 class SubiquityClient {
   late HttpUnixClient _client;
 
   void open(String socketPath) {
+    log.info('Opening socket $socketPath');
     _client = HttpUnixClient(socketPath);
   }
 
   Future<void> close() {
+    log.info('Closing socket ${_client.path}');
     return _client.flush().then((_) => _client.close());
   }
 
   Future<void> checkStatus(String method, StreamedResponse response) async {
     if (response.statusCode != 200) {
       final responseStr = await response.stream.bytesToString();
-      throw ("$method returned error ${response.statusCode}\n$responseStr");
+      log.error('$method returned error ${response.statusCode}\n$responseStr');
```

Comment:
Should the error still be reported to the method caller? Wondering if it's safe to keep sending messages to subiquity if one has unexpectedly failed.